### ISSUE TITLE
inconsistent error message on 'alr config -v'

### ISSUE THREAD

**Issue by RREE** [alire-project/alire#1027]
Using alr v1.2.0-RC1
```
$ alr config -v --list --show-origin --global
-> Using default value for configuration 'distribution.disable_detection': 'FALSE'
-> Spawning: ["uname", "-m"] > GNAT-TEMP-000001.TMP
-> Spawning: ["cat", "/etc/os-release"] > GNAT-TEMP-000002.TMP
error: Unrecognized option '-v' (command/topic "config").
error: try "alr help config" for more information.
```
`-v` is listed as a valid global option for `alr config`, but after three lines of output there is an error message

**Comment by RREE** [comment]
The easiest solution is probably to mention the Global Options in the help output:
```
USAGE
alr [global options] config [options] [--list] [--show-origin] [key_regex] | --get <key> | --set <key> <value> | --unset <key>
```

**Comment by mosteo** [comment]
I see the problem, thanks for the report, @RREE. We have an early parser to activate verbose/debug logging as soon as possible which is failing to verify a change we did in which global options can only come before the subcommand.

**Comment by mosteo** [comment]
Fixed in #1034
